app/src/assets/localization/en/pipette_wizard_flows.json

```
@@ -84,6 +84,8 @@
   "unscrew_and_detach": "Loosen Screws and Detach Mounting Plate",
   "unscrew_at_top": "<block>Loosen the captive screw on the top right of the  carriage. This releases the right pipette mount, which should then freely move up and down.</block>",
   "unscrew_carriage": "unscrew z-axis carriage",
+  "waste_chute_error": "Remove the Waste Chute from the Deck Plate Adaptor before proceeding.",
+  "waste_chute_warning": "If the Waste Chute is installed, remove it from the Deck Plate Adaptor before proceeding.",
```

We've gone with `waste chute` (lowercase) as the short name for the `Opentrons Flex Waste Chute`. 
And all Opentrons products use the `-er` spelling of `adapter`. 

```suggestion
  "waste_chute_error": "Remove the waste chute from the deck plate adapter before proceeding.",
  "waste_chute_warning": "If the waste chute is installed, remove it from the deck plate adapter before proceeding.",
```

Also, if there's space here, we may want to add something like `You don't need to remove the deck plate adapter from slot D3.` Skip it if this is already in crowded/wordy UI.

---

I'll update the casing of these! We're short on space on these screens so I'll leave out the second string.